app/helpers/icons_helper.rb

```diff
@@ -1,6 +1,6 @@
 module IconsHelper
   def graphical_icon(icon, style: nil, css_class: nil, category: 'icons', hex: false, width: nil, height: nil)
-    icon(icon, nil, role: :presentation, style:, css_class:, category:, hex:, width:, height:)
+    icon(icon, nil, alt: '', style:, css_class:, category:, hex:, width:, height:)
```

Comment:
Isn't empty alt text bad?

Reply:
It's the correct way to do icons without semantic meaning (ie things just there for visual purpose, which is what `graphical_icon` is). 

This doesn't actually change anything as we previously passed `nil` as the second param. `role: presentation` isn't actually valid HTML or widly supported by screenreaders (which was the intention when we added it).

That said, this **is** wrong. The second param is the alt text, so this should be:
```suggestion
    icon(icon, "", style:, css_class:, category:, hex:, width:, height:)
```